iidm/iidm-extensions/src/test/java/com/powsybl/iidm/network/extensions/LoadDetailTest.java

```diff
@@ -6,12 +6,18 @@
  */
 package com.powsybl.iidm.network.extensions;
```

Comment:
Shouldn't the test be in package com.powsybl.iidm.network.impl.extensions of iidm-impl?

Reply:
Not sure about it, because it isn't a test specific to the in-memory implementation (it should also work with a persistent for example) however I was not sure if its place was in iidm-tck so I left it here.

Reply:
Ah you're right, but then SlackTerminalTest should not be in iidm-impl either :/